backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/select/concept/ConceptColumnSelect.java

```diff
@@ -0,0 +1,37 @@
+package com.bakdata.conquery.models.datasets.concepts.select.concept;
+
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.models.datasets.concepts.select.Select;
+import com.bakdata.conquery.models.datasets.concepts.tree.TreeConcept;
+import com.bakdata.conquery.models.query.queryplan.aggregators.Aggregator;
+import com.bakdata.conquery.models.query.queryplan.aggregators.specific.value.ConceptElementsAggregator;
+import com.bakdata.conquery.models.query.queryplan.aggregators.specific.value.ConceptValuesAggregator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.dropwizard.validation.ValidationMethod;
+import lombok.Data;
+
+@CPSType(id = "CONCEPT_VALUES", base = Select.class)
+@Data
+public class ConceptColumnSelect extends UniversalSelect {
+
+	private boolean resolved = false;
```

Comment:
Würde ich umbenennen, z.B. in `outputId`. `resolved` heißt im code eigentlich, dass eine id aufgelöst wurde. hier heißt es das man eine Id bekommt. Das ist gut fürs hirntraining :D

Reply:
> resolved heißt im code eigentlich, dass eine id aufgelöst wurde. hier heißt es das man eine Id bekommt. 

Aber hier wird doch auch die resolved-te id ausgegeben, das passt doch zusammen oder?

Was  hälst du von `asIds` ?  Dann steht das da auch in der conf ordentlich drin

Reply:
Das klingt gut :)